FAILED TEST LOG
  ✘ SUBTASKS 1.1 :1 You should rename the "weight" column to "atomic_mass"

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 1 && filtered[0].column_name === 'atomic_mass')

at Context.<anonymous> (test/1.1.test.js:35:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :2 You should have "melting_point_celsius" and "boiling_point_celsius" columns, and not have columns with the old names

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(columns.includes('melting_point_celsius') && columns.includes('boiling_point_celsius') && !columns.includes('melting_point') && !columns.includes('boiling_point'))

at Context.<anonymous> (test/1.1.test.js:48:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :3 Your "melting_point_celsius" and "boiling_point_celsius" columns should not accept null values

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(columns.includes('melting_point_celsius') && columns.includes('boiling_point_celsius'))

at Context.<anonymous> (test/1.1.test.js:61:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :4 You should add the "UNIQUE" constraint to the "symbol" and "name" columns

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 2)

at Context.<anonymous> (test/1.1.test.js:75:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :5 You should add the "NOT NULL" constraint to the suggested columns

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 2)

at Context.<anonymous> (test/1.1.test.js:89:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :6 Your "atomic_number" column should be a foreign key that references the correct column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:103:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :7 You should create a "types" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:116:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :8 Your "types" table should have a "type_id" colunn with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:129:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :9 Your "types" table should have a "type" colunn with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0)

at Context.<anonymous> (test/1.1.test.js:138:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :10 Your "types" table should have the correct three rows inserted

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :11 Your "properties" table should have a "type_id" column with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(filtered.length > 0 && res2.rowCount > 0)

at Context.<anonymous> (test/1.1.test.js:166:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :12 Each row in your "properties" table should have the correct "type_id" value

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :13 You should capitalize the first letter of all "symbol" values

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:188:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :14 You should remove all trailing zero's from value's in the "atomic_mass" column of the "properties" table

error: column "atomic_mass" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :15 You should correctly insert "Fluorine" into the database

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :16 You should correctly insert "Neon" into the database

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :17 Your "periodic_table" folder should be a git repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :18 Your repository should have a main branch

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :19 Your repository should have at least five commits

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:247:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :20 You should create an "element.sh" file in your repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :21 Your "element.sh" file should have executable permissions

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/1.1.test.js:263:32)
  ✘ SUBTASKS 1.1 :22 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :23 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :24 Running your script with a random existing element gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :25 Running your script with input that doesn't exist should give the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :26 Your first commit message should be "Initial commit"

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:303:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :27 All your commit messages except the first should start with one of the suggested prefixes

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:311:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :28 You should delete the rows of the non-existent element from your tables

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:331:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :29 Your "properties" table should not have a "type" column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:340:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :30 You should finish on the "main" branch, your working tree should be clean and there should not be any uncommitted changes

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
FAILED TEST LOG
  ✘ SUBTASKS 1.1 :2 You should have "melting_point_celsius" and "boiling_point_celsius" columns, and not have columns with the old names

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(columns.includes('melting_point_celsius') && columns.includes('boiling_point_celsius') && !columns.includes('melting_point') && !columns.includes('boiling_point'))

at Context.<anonymous> (test/1.1.test.js:48:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :3 Your "melting_point_celsius" and "boiling_point_celsius" columns should not accept null values

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(columns.includes('melting_point_celsius') && columns.includes('boiling_point_celsius'))

at Context.<anonymous> (test/1.1.test.js:61:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :4 You should add the "UNIQUE" constraint to the "symbol" and "name" columns

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 2)

at Context.<anonymous> (test/1.1.test.js:75:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :5 You should add the "NOT NULL" constraint to the suggested columns

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 2)

at Context.<anonymous> (test/1.1.test.js:89:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :6 Your "atomic_number" column should be a foreign key that references the correct column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:103:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :7 You should create a "types" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:116:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :8 Your "types" table should have a "type_id" colunn with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:129:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :9 Your "types" table should have a "type" colunn with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0)

at Context.<anonymous> (test/1.1.test.js:138:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :10 Your "types" table should have the correct three rows inserted

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :11 Your "properties" table should have a "type_id" column with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(filtered.length > 0 && res2.rowCount > 0)

at Context.<anonymous> (test/1.1.test.js:166:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :12 Each row in your "properties" table should have the correct "type_id" value

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :13 You should capitalize the first letter of all "symbol" values

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:188:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :14 You should remove all trailing zero's from value's in the "atomic_mass" column of the "properties" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:201:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :15 You should correctly insert "Fluorine" into the database

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :16 You should correctly insert "Neon" into the database

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :17 Your "periodic_table" folder should be a git repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :18 Your repository should have a main branch

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :19 Your repository should have at least five commits

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:247:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :20 You should create an "element.sh" file in your repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :21 Your "element.sh" file should have executable permissions

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/1.1.test.js:263:32)
  ✘ SUBTASKS 1.1 :22 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :23 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :24 Running your script with a random existing element gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :25 Running your script with input that doesn't exist should give the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :26 Your first commit message should be "Initial commit"

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:303:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :27 All your commit messages except the first should start with one of the suggested prefixes

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:311:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :28 You should delete the rows of the non-existent element from your tables

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:331:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :29 Your "properties" table should not have a "type" column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:340:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :30 You should finish on the "main" branch, your working tree should be clean and there should not be any uncommitted changes

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
FAILED TEST LOG
  ✘ SUBTASKS 1.1 :7 You should create a "types" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:116:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :8 Your "types" table should have a "type_id" colunn with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:129:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :9 Your "types" table should have a "type" colunn with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0)

at Context.<anonymous> (test/1.1.test.js:138:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :10 Your "types" table should have the correct three rows inserted

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :11 Your "properties" table should have a "type_id" column with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(filtered.length > 0 && res2.rowCount > 0)

at Context.<anonymous> (test/1.1.test.js:166:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :12 Each row in your "properties" table should have the correct "type_id" value

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :13 You should capitalize the first letter of all "symbol" values

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:188:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :14 You should remove all trailing zero's from value's in the "atomic_mass" column of the "properties" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:201:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :15 You should correctly insert "Fluorine" into the database

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :16 You should correctly insert "Neon" into the database

error: relation "types" does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :17 Your "periodic_table" folder should be a git repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :18 Your repository should have a main branch

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :19 Your repository should have at least five commits

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:247:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :20 You should create an "element.sh" file in your repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :21 Your "element.sh" file should have executable permissions

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/1.1.test.js:263:32)
  ✘ SUBTASKS 1.1 :22 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :23 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :24 Running your script with a random existing element gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :25 Running your script with input that doesn't exist should give the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :26 Your first commit message should be "Initial commit"

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:303:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :27 All your commit messages except the first should start with one of the suggested prefixes

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:311:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :28 You should delete the rows of the non-existent element from your tables

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:331:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :29 Your "properties" table should not have a "type" column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:340:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :30 You should finish on the "main" branch, your working tree should be clean and there should not be any uncommitted changes

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
FAILED TEST LOG
  ✘ SUBTASKS 1.1 :10 Your "types" table should have the correct three rows inserted

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(types.includes('metal') && types.includes('metalloid') && types.includes('nonmetal'))

at Context.<anonymous> (test/1.1.test.js:151:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :11 Your "properties" table should have a "type_id" column with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(filtered.length > 0 && res2.rowCount > 0)

at Context.<anonymous> (test/1.1.test.js:166:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :12 Each row in your "properties" table should have the correct "type_id" value

error: column properties.type_id does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :13 You should capitalize the first letter of all "symbol" values

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:188:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :14 You should remove all trailing zero's from value's in the "atomic_mass" column of the "properties" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:201:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :15 You should correctly insert "Fluorine" into the database

error: column properties.type_id does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :16 You should correctly insert "Neon" into the database

error: column properties.type_id does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :17 Your "periodic_table" folder should be a git repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :18 Your repository should have a main branch

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :19 Your repository should have at least five commits

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:247:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :20 You should create an "element.sh" file in your repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :21 Your "element.sh" file should have executable permissions

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/1.1.test.js:263:32)
  ✘ SUBTASKS 1.1 :22 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :23 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :24 Running your script with a random existing element gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :25 Running your script with input that doesn't exist should give the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :26 Your first commit message should be "Initial commit"

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:303:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :27 All your commit messages except the first should start with one of the suggested prefixes

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:311:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :28 You should delete the rows of the non-existent element from your tables

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:331:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :29 Your "properties" table should not have a "type" column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:340:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :30 You should finish on the "main" branch, your working tree should be clean and there should not be any uncommitted changes

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
FAILED TEST LOG
  ✘ SUBTASKS 1.1 :11 Your "properties" table should have a "type_id" column with the correct properties

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(filtered.length > 0 && res2.rowCount > 0)

at Context.<anonymous> (test/1.1.test.js:166:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :12 Each row in your "properties" table should have the correct "type_id" value

error: column properties.type_id does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :13 You should capitalize the first letter of all "symbol" values

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:188:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :14 You should remove all trailing zero's from value's in the "atomic_mass" column of the "properties" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:201:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :15 You should correctly insert "Fluorine" into the database

error: column properties.type_id does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :16 You should correctly insert "Neon" into the database

error: column properties.type_id does not exist
at Parser.parseErrorMessage (node_modules/pg-protocol/dist/parser.js:278:15)
at Parser.handlePacket (node_modules/pg-protocol/dist/parser.js:126:29)
at Parser.parse (node_modules/pg-protocol/dist/parser.js:39:38)
at Socket.<anonymous> (node_modules/pg-protocol/dist/index.js:8:42)
at Socket.emit (node:events:518:28)
at addChunk (node:internal/streams/readable:559:12)
at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
at Readable.push (node:internal/streams/readable:390:5)
at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
  ✘ SUBTASKS 1.1 :17 Your "periodic_table" folder should be a git repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :18 Your repository should have a main branch

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :19 Your repository should have at least five commits

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:247:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :20 You should create an "element.sh" file in your repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :21 Your "element.sh" file should have executable permissions

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/1.1.test.js:263:32)
  ✘ SUBTASKS 1.1 :22 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :23 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :24 Running your script with a random existing element gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :25 Running your script with input that doesn't exist should give the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :26 Your first commit message should be "Initial commit"

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:303:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :27 All your commit messages except the first should start with one of the suggested prefixes

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:311:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :28 You should delete the rows of the non-existent element from your tables

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:331:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :29 Your "properties" table should not have a "type" column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:340:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :30 You should finish on the "main" branch, your working tree should be clean and there should not be any uncommitted changes

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
FAILED TEST LOG
  ✘ SUBTASKS 1.1 :13 You should capitalize the first letter of all "symbol" values

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:188:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :14 You should remove all trailing zero's from value's in the "atomic_mass" column of the "properties" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:201:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :15 You should correctly insert "Fluorine" into the database

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:214:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :16 You should correctly insert "Neon" into the database

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:227:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :17 Your "periodic_table" folder should be a git repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :18 Your repository should have a main branch

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :19 Your repository should have at least five commits

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:247:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :20 You should create an "element.sh" file in your repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :21 Your "element.sh" file should have executable permissions

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/1.1.test.js:263:32)
  ✘ SUBTASKS 1.1 :22 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :23 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :24 Running your script with a random existing element gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :25 Running your script with input that doesn't exist should give the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :26 Your first commit message should be "Initial commit"

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:303:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :27 All your commit messages except the first should start with one of the suggested prefixes

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:311:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :28 You should delete the rows of the non-existent element from your tables

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:331:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :29 Your "properties" table should not have a "type" column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:340:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :30 You should finish on the "main" branch, your working tree should be clean and there should not be any uncommitted changes

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
FAILED TEST LOG
  ✘ SUBTASKS 1.1 :14 You should remove all trailing zero's from value's in the "atomic_mass" column of the "properties" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:201:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :15 You should correctly insert "Fluorine" into the database

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:214:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :16 You should correctly insert "Neon" into the database

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:227:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :17 Your "periodic_table" folder should be a git repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :18 Your repository should have a main branch

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :19 Your repository should have at least five commits

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:247:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :20 You should create an "element.sh" file in your repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :21 Your "element.sh" file should have executable permissions

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/1.1.test.js:263:32)
  ✘ SUBTASKS 1.1 :22 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :23 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :24 Running your script with a random existing element gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :25 Running your script with input that doesn't exist should give the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :26 Your first commit message should be "Initial commit"

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:303:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :27 All your commit messages except the first should start with one of the suggested prefixes

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:311:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :28 You should delete the rows of the non-existent element from your tables

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:331:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :29 Your "properties" table should not have a "type" column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:340:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :30 You should finish on the "main" branch, your working tree should be clean and there should not be any uncommitted changes

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
FAILED TEST LOG
  ✘ SUBTASKS 1.1 :14 You should remove all trailing zero's from value's in the "atomic_mass" column of the "properties" table

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length === 0)

at Context.<anonymous> (test/1.1.test.js:201:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :15 You should correctly insert "Fluorine" into the database

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:214:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :16 You should correctly insert "Neon" into the database

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount > 0 && filtered.length > 0)

at Context.<anonymous> (test/1.1.test.js:227:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :17 Your "periodic_table" folder should be a git repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :18 Your repository should have a main branch

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :19 Your repository should have at least five commits

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:247:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :20 You should create an "element.sh" file in your repository

Error: ENOENT: no such file or directory, scandir '../periodic_table'
  ✘ SUBTASKS 1.1 :21 Your "element.sh" file should have executable permissions

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(false)

at Context.<anonymous> (test/1.1.test.js:263:32)
  ✘ SUBTASKS 1.1 :22 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :23 Running your script with the suggested input gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :24 Running your script with a random existing element gives the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :25 Running your script with input that doesn't exist should give the output described

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  ✘ SUBTASKS 1.1 :26 Your first commit message should be "Initial commit"

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:303:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :27 All your commit messages except the first should start with one of the suggested prefixes

Error: The path ../periodic_table was not found
at validatePath (node_modules/git-parse/dist/util.js:27:11)
at gitToJs (node_modules/git-parse/dist/index.js:44:28)
at getCommits (test/utils.js:166:25)
at Context.<anonymous> (test/1.1.test.js:311:27)
at process.processImmediate (node:internal/timers:478:21)
  ✘ SUBTASKS 1.1 :28 You should delete the rows of the non-existent element from your tables

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:331:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :29 Your "properties" table should not have a "type" column

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(res.rowCount === 0)

at Context.<anonymous> (test/1.1.test.js:340:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  ✘ SUBTASKS 1.1 :30 You should finish on the "main" branch, your working tree should be clean and there should not be any uncommitted changes

Error: spawn /bin/bash ENOENT
at ChildProcess._handle.onexit (node:internal/child_process:286:19)
at onErrorNT (node:internal/child_process:484:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
